screens/Notifications/NotificationScreen.js

```diff
@@ -28,31 +29,33 @@ function NotificationScreen() {
   return (
     <>
       <Header title='Notificationes' />
```

Comment:
Esto lo pifie yo pero me acabo de dar cuenta, lo puedes arreglar porfa?
```suggestion
      <Header title='Notificaciones' />
```

Reply:
Sii, tampoco lo habia visto